### ISSUE TITLE
Copyright text under "About Xamarin Inspector" Window is not having color contrast ratio of 4.5:1.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#281]
### Steps to Reproduce
1. Launch Visual studio and Sign.
2. Create a project and build it.
3. Now Launch Xamarin Inspector from visual studio.
4. Navigate to Xamarin Inspector menu item and invoke "About Xamarin Inspector" pop window.
5. Observe the color contrast ratio of Copy right text under version of Xamarin Inspector popup.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1.

### Actual Behavior
Color contrast ratio is less than 4.5:1 (i.e., 4.1:1) as shown in the attached screen-shot.

### User Impact
Low vision user will feel difficulty in seeing the text.

![issue 5](https://user-images.githubusercontent.com/22956130/39869203-4d2f2564-547a-11e8-8452-5b4fe0c5a21f.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro 
MAS1.4.3

**Comment by abock** [comment]
Closing as a duplicate of #278.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
